From: Adrian Bunk <bunk@stusta.de>
Subject: [2.6 patch] add -Werror-implicit-function-declaration to CFLAGS
Patch-mainline: breaks akpm, but helps SuSE

Currently, using an undeclared function gives a compile warning, but it 
can lead to a link or even a runtime error.

With -Werror-implicit-function-declaration, we are getting an immediate 
compile error instead.

This patch also removes some unneeded spaces between two tabs in the 
following line.


Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Olaf Hering <olh@suse.de>

---
 Makefile |    1 +
 1 file changed, 1 insertion(+)

--- a/Makefile	2007-08-27 14:01:19.000000000 -0400
+++ b/Makefile	2007-08-27 14:04:32.000000000 -0400
@@ -313,6 +313,7 @@ LINUXINCLUDE    := -Iinclude \
 CPPFLAGS        := -D__KERNEL__ $(LINUXINCLUDE)
 
 CFLAGS          := -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs \
+                   -Werror-implicit-function-declaration \
                    -fno-strict-aliasing -fno-common
 AFLAGS          := -D__ASSEMBLY__
 
